README.md

```diff
@@ -127,6 +127,24 @@ Download specific tests based on their id on the Rainforest dashboard
 rainforest download 33445 11232 1337
 ```
 
+#### Running Local RFML Tests Only
+
+If you want to run a local set of RFML files (for instance in a CI environment), use the `run -f` option:
+
+```
+rainforest run -f [FILES OR FOLDERS]
+```
+
+`run -f` accepts any number of files and folders as arguments (folders will be recursively checked for `*.rfml` files). All embedded tests must be included within `FILES OR FOLDERS`.
+
+There is a specific metadata option in RFML files for `run -f`: `# execute: true|false`, which indicates whether a test should be run by default (defaults to `true`). Embedded tests that are not usually run directly should specify `# execute: false`.
+
+Most options for `rainforest run` work the same way with `run -f`, with the exception of test filtering options (such as `--folder`). In addition, there are a few options that are specific to `run -f` or behave differently:
```

Comment:
This seems ambiguous. Are there any other options that do not work with `-f` or is it only the filtering options that work differently/don't work at all?